internal/certgen/certgen.go

```diff
@@ -103,6 +104,10 @@ func writeCACertKube(client *kubernetes.Clientset, namespace string, cert []byte
 	secret := newCertOnlySecret("cacert", namespace, "cacert.pem", cert)
 	_, err := client.CoreV1().Secrets(namespace).Create(secret)
 	if err != nil {
+		if k8serrors.IsAlreadyExists(err) {
```

Comment:
Nit: would this style be more canonical?
```Go
if _, err := client.CoreV1().Secrets(namespace).Create(secret); if err != nil {
        ...
}
```

Reply:
Won't hurt. I'll change it.